protocol/proofs/delayed-weth-expansion.md

```diff
@@ -0,0 +1,62 @@
+subject to the Superchain-wide pause mechanism within the `SuperchainConfig`. Bonds cannot be
+retrieved by game participants if the Superchain pause is active.
+
+## Problem Statement
+
+Existing security mechanisms within the `DelayedWETH` contract are inconvenient and slow down the
+incident response process. Specifically:
+
+- The `DelayedWETH.hold(address target)` function creates an approval from the `target` address to
+  the `owner` address. Since the `target` can simply remove this approval, `hold` MUST be triggered
+  alongside a `transferFrom` in the same transaction. This adds unnecessary complexity to something
+  that could simply be a single transaction.
+- The various security mechanisms inside of `DelayedWETH` can only be triggered by the System Owner
+  which has a slow SLA. The Deputy Guardian can act to trigger the Superchain-wide pause, but this
+  is highly disruptive to the entire Superchain ecosystem.
+
+Additionally, it should be noted that all ETH inside a `DelayedWETH` contract is pooled together
+regardless of which games are using the contract. We currently mitigate this by using one
+`DelayedWETH` contract per game type, but this expands the number of contracts that the System
+Owner and Guardian roles must manage by at least 2 per additional chain. However, the modifications
+required to appropriately address this issue are sufficiently extensive that they are considered
+out of scope for this proposal.
+
+## Propsed Solution
+
+Here we propose some modifications to `DelayedWETH` that help to resolve the two noted limitations:
+
+1. We propose that `DelayedWETH.hold` executes a `transferFrom` automatically to reduce the number
+  of required transactions from 2 to 1 when holding ETH from a single dispute game.
+2. We propose a new `pause` function on the `DelayedWETH` contract itself that pauses *just* the
+  `DelayedWETH` contract and supplements the Superchain-wide pause functionality. This `pause`
+  function would be executable by the Guardian role and could therefore be triggered quickly in
+  case the System Owner is unable to execute another security action in time.
+
+## Alternatives Considered
+
+### Wider Rewrite
+
+We considered the alternative to rewrite the `DelayedWETH` contract more broadly to be able to
+easily manage multiple game types across multiple chains within a single `DelayedWETH` instance.
+However, this would require significant modifications to the `DelayedWETH` contract and would act
+as an iterative improvement to the status quo. We still believe that a larger rewrite should be
+considered for the near future.
+
+## Risks & Uncertainties
+
+### Security Model
+
+We see little risk with the modification to the `hold` function. Introducing a new `pause` function
+slightly changes the existing security model (though not significantly). The Guardian role can
+already pause the `DelayedWETH` contract by triggering the Superchain-wide pause. Allowing this
```

Comment:
> The Guardian role can already pause the `DelayedWETH` contract by triggering the Superchain-wide pause

What do you mean by this? The Superchain-wide pause only pauses all withdrawals (ETH, ERC20, ERC721, and arbitrary calls) from the bridge contracts, but it does not affect the DelayedWETH contract. So I'm assuming you mean that it effectively indirectly pauses DelayedWETH because a withdrawal cannot be executed until unpaused? (I am not too familiar with the current DelayedWETH mechanics and usage)

Reply:
The Superchain-wide pause affects DelayedWETH contract too, see here https://github.com/ethereum-optimism/optimism/blob/6f41bac618826b7669ba584876586fb17ac5e9b0/packages/contracts-bedrock/src/dispute/DelayedWETH.sol#L92